tGUI.m

```diff
@@ -74,7 +74,87 @@
 %                      - saved function as tGUI.m 
 % 2017.12.05 - coolbac - updated to version 0.2.2 changes include:
 %		       - corrected errors in version 0.2.1 that did not save during previous commit
-function csv_filename = tGUI(experiment_name_string, gui_params)
+% 2018.07.19 - joseferpaz - updated to version 1.0.0 changes include:
+%                         - compatibility with arduino temperature sensor
+%                         - temperature readings logged into csv file
+%                         - Waits for thermoneutral before running tGUI
+%                     
+%
+%% Initialize arduino
+
+clear all;
+global sensor1_analog sensor1_t sensor2_analog sensor2_t uno;
+global sensor1_port sensor2_port filename;
+
+uno  = arduino();
+  
+%Port for LED
+light_port = 'D3';
+  
+%Port for sensor 1 (Green)
```

Comment:
Is it possible for other ports on the Arduino to be used for the LED, and temperature sensors? If so, we could include a comment to indicate the use of any digital port. 

Related to this idea of increased flexibility, we might want to take a picture of the Arduino and possibly create a simple wiring diagram (the program Fritzing is good) to include with the code. We can also create an equipment/parts list in Excel so others now what and where to buy the hardware.